CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### New features
 
 ### Fixes and improvements
+* Replace `boost::program_options` with `cxxopts` for client options
```

Comment:
This looks like an old change. Could you clean up the branch?

Reply:
Done (hopefully) :).